### PR TITLE
zippo: Initial crDroid 10

### PR DESCRIPTION
**Pull request by chematelegram**

### CODE REVIEW

File: zippo.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        "device": "Lenovo Z6 Pro",
+        "filename": "crDroidAndroid-14.0-20240115-zippo-v10.1.zip",
+        "download": "https://sourceforge.net/projects/crdroid/files/zippo/10.x/crDroidAndroid-14.0-20240115-zippo-v10.1.zip/download",
+        "timestamp": 1705278688,
+        "md5": "536a68e5c8d8e7890949d319d3555297",
+        "sha256": "61296f6aa43d381547a1d8f472c850d79087d2d49c666fbb4bd5d97f3ec1b051",
+        "size": 1044908326,
+        "version": "10.1",
+        "buildtype": "Monthly",
+        "forum": "https://xdaforums.com/t/rom-14-official-retrofit-zippo-crdroid-v10-x.4651042/",
```
Comment (ghost) [inline zippo.json:15]: Fix forum post.
Screenshot_20240115-175048_Firefox
Must be "https://github.com/crdroidandroid/crdroid_features/blob/14.0/README.mkdn"
Screenshot_20240115-175112_Firefox
Wrong download link.
Changelog link must be "https://crdroid.net/zippo/10#changelog"

**Review by ghost — CHANGES_REQUESTED** [review]

File: zippo.json (hunk 1 above)
Comment (gwolf2u) [reply zippo.json:15]: also, missing follow-back donation url

File: zippo.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        "gapps": "https://sourceforge.net/projects/nikgapps/files/Config-Releases/NikGapps-U/14-Jan-2024/NikGapps-chematelegram-arm64-14-20240114.zip/download",
```
Comment (gwolf2u) [inline zippo.json]: tried our gapps?
https://nikgapps.com/crdroid-official

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: zippo.json (hunk 1 above)
Comment (chematelegram) [reply zippo.json:15]: Thanks for the detailed observations, I have edited all of them. Thanks in advance for the new review.

**Review by gwolf2u — APPROVED** [review]

**Pull request merged**